Formula 1/Views/Cells/CircuitCell.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  CircuitCell.swift
+//  Formula 1
+//
+//  Created by Gio on 11/13/19.
+//  Copyright © 2019 Gio. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class CircuitCell: UICollectionViewCell {
+    static let reuseIdentifier = String(describing: self)
+    
+    lazy var nameLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.adjustsFontForContentSizeCategory = true
+        label.font = UIFont.Formula1Font.Regular
+        return label
+    }()
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        
+        addSubviews()
+        activateConstraints()
+        applySkin()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    func addSubviews() {
+        contentView.addSubview(nameLabel)
+    }
+    
+    func activateConstraints() {
+        NSLayoutConstraint.activate([
+            
+            nameLabel.centerXAnchor.constraint(equalTo: contentView.centerXAnchor),
+            nameLabel.centerYAnchor.constraint(equalTo: contentView.centerYAnchor)
+        ])
+    }
```

Comment:
Same issue with crashing. Also, do they need to be public?

Reply:
sorry i dont understand. the anchors?

Reply:
See https://github.com/gionoa/Formula-1/pull/4/files#r346049715